Pipfile

```diff
@@ -27,4 +27,4 @@ azafea = "python -m azafea"
 doc = "sphinx-build -b html docs/source docs/build/html"
 lint = "py.test --flake8 --mypy -m 'flake8 or mypy'"
 test = "py.test --cov=azafea --cov-fail-under=86 --no-cov-on-fail -m 'not integration'"
-test-all = "py.test --cov=azafea --cov-fail-under=99 --no-cov-on-fail"
+test-all = "py.test --cov=azafea --cov-fail-under=99 --no-cov-on-fail --verbose"
```

Comment:
Always nice when trying to debug something causes the problem to go away :-)

The output with this flag is not too verbose so let's leave it on.